iothub/device/src/InternalClient.cs

```diff
@@ -311,9 +328,16 @@ public async Task AbandonMessageAsync(string lockToken, CancellationToken cancel
         }
 
         /// <summary>
-        /// Puts a received message back onto the device queue
+        /// Puts a received message back onto the client queue.
         /// </summary>
-        /// <returns>The lock identifier for the previously received message</returns>
+        /// <remarks>
+        /// You cannot reject or abandon messages over MQTT protocol. For more details, see
```

Comment:
Prefer non-personal documentation, such as:

```suggestion
        /// Messages cannot be rejected or abandoned over the MQTT protocol. For more details, see
```